gruntfile.js

```diff
@@ -92,13 +92,13 @@ module.exports = function(grunt) {
           'dist/popup.min.js': [
             pathLibs + 'jquery/dist/jquery.min.js',
             pathScripts + 'chrome-service.js',
-            pathScripts + 'usd-value-service.js',
+            pathScripts + 'currency-service.js',
             pathScripts + 'popup.js'
           ],
           'dist/notifier.min.js': [
```

Comment:
Maybe there is no more sense in name it as `notifier`, once it holds much more than just the notifier service.
What d'you think about `background.min.js`. So we'll have `popup.min.js` and `background.min.js`. Seems to make sense for me.